README.md

```diff
@@ -7,6 +7,36 @@ Trojstenovy web
 [![codecov](https://codecov.io/gh/trojsten/web/branch/master/graph/badge.svg?token=t4kSkwFccG)](https://codecov.io/gh/trojsten/web)
 ![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)
 
+## Getting started
+
+This project uses [pre-commit](https://pre-commit.com/). 
+You can install it using `pip install pre-commit` and it's also included in [the project requirements](https://github.com/trojsten/web/blob/master/requirements.txt).
+
+Activate it with 
+```bash
+pre-commit install --allow-missing-config
+```
+
+### Running localy
+
+Please follow the instalation manual here: https://github.com/trojsten/web/wiki/Návod-na-inštaláciu
+
+### Commiting code
+
+Please send a pullrequest with descriptive title and all necessary information in the description. Preferably all in english.
+The pullrequest needs to pass automatic checks and have at least one approval from the project maintainters.
+
+We currently check following things:
+- code style
+  - the code must follow [black](https://github.com/python/black) codestyle. The easiest way to achieve this is automatically format with the `black` tool. This is also enforced by the pre-commit check. Note that black is currently not included in the project requirements, because it requires Python 3.6+.
```

Comment:
```suggestion
  - the code must follow [black](https://github.com/python/black) codestyle. The easiest way to achieve this is to automatically format the code with the `black` tool. This is also enforced by the pre-commit check. Note that black is currently not included in the project requirements, because it requires Python 3.6+.
```